src/shared/styles/core/core.scss

```diff
@@ -8,8 +12,8 @@
 
 @import '~semantic-ui-sass/scss/collections/grid';
 
-body {
-    margin: 0;
-    padding: 0;
-    font-family: sans-serif;
+
+.sb-show-main {
```

Comment:
Reset styles are breaking the storybook layout/background so we need this as a temp fix. I'll look into stopping the `body` styles from being overwritten by our reset.